src/common/btree.c

```diff
@@ -1230,7 +1244,7 @@ btr_root_resize(struct btr_context *tcx, struct btr_trace *trace,
 		return rc;
 	}
 	trace->tr_node = root->tr_node = nd_off;
-	memcpy(btr_off2ptr(tcx, nd_off), nd, old_size);
+	umem_tx_memcpy(btr_umm(tcx), btr_off2ptr(tcx, nd_off), nd, old_size);
```

Comment:
(style)  'nd' may be misspelled - perhaps 'and'?
(style)  'nd' may be misspelled - perhaps 'and'?